__mocks__/user/failed-login/POST.mock

```diff
@@ -0,0 +1,16 @@
+HTTP/1.1 401 Unauthorized
+Content-Type: application/json
+
+{{#code}}
+(() => {
+    return {
+        status: 401,
+        headers: {
+            'X-Requested-By': 'login/user',
+        },
+        body: JSON.stringify({
+            message: "Неправильний email або пароль"
+        })
+    };
+})();
+{{/code}}
```

Comment:
please move all mocks under the `__mock__/v1`

Reply:
done